### ISSUE TITLE
setup-autotools.sh returns permission denied error

### ISSUE THREAD

**Issue by kper** [sandstorm-io/capnproto#477]
Hi,

I am having a capnproto dependency (branch 0.5.3) and unfortunately my travis build have begun to fail. 

Output:
```
$ ./setup-autotools.sh

================================================================================

Fetching Google Test code...

================================================================================

Cloning into 'gtest'...

Warning: Permanently added the RSA host key for IP address 'xxx.xxx.xxx.xxx' to the list of known hosts.

Permission denied (publickey).

fatal: Could not read from remote repository.

Please make sure you have the correct access rights

and the repository exists.
```

**Comment by dwrensha** [comment]
I wonder if this is related to https://github.com/sandstorm-io/capnproto/issues/388.

**Comment by kentonv** [comment]
@kper Oops, I recently changed setup-autotools.sh to fetch gtest from github because it was previously trying to fetch from Google Code, which is long-gone. But, apparently I made it fetch over SSH when I should have used HTTPS, so it worked locally but wouldn't work on machines that don't have a Github SSH key installed.

I just pushed 983878f8 which should fix this. Sorry for the trouble!

That said I'm surprised your build was passing before since Google Code has been dead for a long time now.

On another note, 0.6.0 is out now. You might want to consider upgrading. :)
